react.html

```diff
@@ -67,13 +67,13 @@
   	  },
   	  timerTick: function() {
   	  	this.setState({
-  	  		strikes: this.state.strikes + 100
+  	  		strikes: this.state.strikes + 0
```

Comment:
I wanted it to stop, while I worked on something else,  it was bugging me.  I know it's weird, but it just was.